formbricks/formbricks
# fix: form rerender on name change

## What does this PR do?


1. Fixes the form preview re-render on name change.
1.1. This issue was caused by unnecessary Resetting question progress on name change.

Fixes # (issue)
Fixed: #968 



## Type of change




## How should this be tested?



- Test A
   - Try changing name while editing form, notice for re-render in the form preview. 
- Test B
  - Implement a method or change name as fast as possible to check if preview is still breaking. 

## Checklist



### Required


### Appreciated
